packages/edit-widgets/src/components/layout/index.js

```diff
@@ -23,6 +24,7 @@ function Layout( { blockEditorSettings } ) {
 	return (
 		<SlotFillProvider>
 			<DropZoneProvider>
+				<BlockEditorKeyboardShortcuts.Register />
```

Comment:
Do we include `<BlockEditorKeyboardShortcuts :>` elsewhere on this screen? (we need both the registration and the application of  the shortcuts to make them work)

Reply:
Yes, we already were applying them here https://github.com/WordPress/gutenberg//blob/cc378e8f09a1d74bc8982e453a532e8c78d52225/packages/edit-widgets/src/components/widget-area/index.js#L127. Before there was no need to register them and things worked now, this PR adds the registering, so the shortcuts continue to work.